deployment/build-s3-dist.sh

```diff
@@ -46,58 +54,32 @@ echo "--------------------------------------------------------------------------
 echo "[Init] Remove any old dist files from previous runs"
```

Comment:
duplicate set -x
recommend set -o pipefail as well

Reply:
updated to include a full `set -eou pipefail` immediately after parsing input variables